src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/MongoDb26/Instrumentation.xml

```diff
@@ -37,86 +38,115 @@ SPDX-License-Identifier: Apache-2.0
       </match>
     </tracerFactory>
 
-  <tracerFactory name="AsyncCursorWrapper">
-    <match assemblyName="MongoDB.Driver.Core" className="MongoDB.Driver.Core.Operations.AsyncCursor`1">
-    <exactMethodMatcher methodName="GetNextBatch"/>
-    <exactMethodMatcher methodName="GetNextBatchAsync"/>
-    </match>
-  </tracerFactory>
+    <tracerFactory name="AsyncCursorWrapper">
+      <match assemblyName="MongoDB.Driver.Core" className="MongoDB.Driver.Core.Operations.AsyncCursor`1" maxVersion="2.999.0">
```

Comment:
`maxVersion` is exclusive, so you can set it to `3.0.0` safely.

Reply:
```suggestion
      <match assemblyName="MongoDB.Driver.Core" className="MongoDB.Driver.Core.Operations.AsyncCursor`1" maxVersion="3.0.0">
```

Reply:
I always forget that, thanks for catching it!